tests/vm_add_nic

```diff
@@ -0,0 +1,84 @@
+clear cc responses
+vm config kernel $images/miniccc.kernel
+vm config initrd $images/miniccc.initrd
+# Launch a vm with no network
+vm launch kvm foo
+
+# Launch one vm on vlan 100
+vm config net 100
+vm launch kvm bar
+vm start all
+
+# wait for cc to connect
+shell sleep 30
+
+.column name,vlan,bridge,ip vm info
+
+# try adding a nic to foo on the same vlan as bar
+vm net add foo 100
```

Comment:
Probably should add line to test the full command, vm net add <target> <bridge,vlan,mac,driver>
"vm net add foo test,100,3c:d9:ce:82:00:88"

make sure both vm nics are on same bridge